papers/ian_kenney/ian_kenney.rst

```diff
@@ -457,7 +457,7 @@ This document is just the first step and broad guide to our vision of developing
 Acknowledgments
 ~~~~~~~~~~~~~~~
 
-We gratefully acknowledge the 184 developers and countless community members who have contributed to the MDAnalysis project over the last 16 years and NumFOCUS for its support as our fiscal sponsor.
+We gratefully acknowledge the 184 developers and countless community members who have contributed to the MDAnalysis project since its inception and NumFOCUS for its support as our fiscal sponsor.
```

Comment:
I woke up too early to figure out if we have 18 or 19 years development (or getting into 20?), so the wording here has been changed.